lib/core/test/unit/Cardano/Wallet/Primitive/TypesSpec.hs

```diff
@@ -318,25 +318,34 @@ spec = do
             let slotParams = mockSlotParams 0.5
             let ntwkTip = SlotId 1 0
             let plots =
-                    [ (mockBlockHeader (SlotId 0 0) 0, 0)
-                    , (mockBlockHeader (SlotId 0 1) 1, 20)
-                    , (mockBlockHeader (SlotId 0 2) 2, 33)
-                    , (mockBlockHeader (SlotId 0 3) 2, 33)
-                    , (mockBlockHeader (SlotId 0 4) 2, 40)
-                    , (mockBlockHeader (SlotId 0 5) 3, 60)
-                    , (mockBlockHeader (SlotId 0 6) 3, 60)
-                    , (mockBlockHeader (SlotId 0 7) 4, 66)
-                    , (mockBlockHeader (SlotId 0 8) 5, 83)
-                    , (mockBlockHeader (SlotId 0 9) 5, 100)
-                    , (mockBlockHeader (SlotId 1 0) 5, 100)
+                    [ (mockBlockHeader (SlotId 0 0) 0, 0.0)
+                    , (mockBlockHeader (SlotId 0 1) 1, 0.2)
+                    , (mockBlockHeader (SlotId 0 2) 2, 0.3333)
+                    , (mockBlockHeader (SlotId 0 3) 2, 0.3333)
```

Comment:
Here we see the effect of rounding `remainingBlocks`!

These should be the numbers from the un-rounded `remainingBlocks`:
```
[ (mockBlockHeader (SlotId 0 0) 0, 0.0)
, (mockBlockHeader (SlotId 0 1) 1, 0.1818)
, (mockBlockHeader (SlotId 0 2) 2, 0.3333)
, (mockBlockHeader (SlotId 0 3) 2, 0.3636)
, (mockBlockHeader (SlotId 0 4) 2, 0.40)
, (mockBlockHeader (SlotId 0 5) 3, 0.5455)
, (mockBlockHeader (SlotId 0 6) 3, 0.60)
, (mockBlockHeader (SlotId 0 7) 4, 0.7273)
, (mockBlockHeader (SlotId 0 8) 5, 0.8333)
, (mockBlockHeader (SlotId 0 9) 5, 0.9091)
, (mockBlockHeader (SlotId 1 0) 5, 1.00)
```
Notice 0.3333 and 0.3636 vs 0.3333 and 0.3333

Reply:
Indeed, on a chain of length ~5, which already gives satisfying results regarding progress tracking. Don't you think?